### PR TITLE
home page

### PR DESCRIPTION
**Pull request by prakashsakari**
This PR has the following
1. Landing Page with different quiz categories

quiz-home

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import url("https://uilight.netlify.app/main.css");
+@import url("./components/Navbar/Navbar.css");
+@import url("./components/QuizCard/QuizCard.css");
+@import url("./components/Footer/Footer.css");
+:root {
```
Comment (rohanmathur91) [inline src/App.css]: have a spacing

File: src/components/QuizCard/QuizCard.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import "./QuizCard.css";
+export const QuizCard = ({ category }) => {
```
Comment (rohanmathur91) [inline src/components/QuizCard/QuizCard.jsx]: have a spacing

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import {BrowserRouter} from "react-router-dom";
+ReactDOM.render(
```
Comment (rohanmathur91) [inline src/index.js]: same as above

File: src/reportWebVitals.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```
Comment (rohanmathur91) [inline src/reportWebVitals.js]: you can delete this file

File: src/setupTests.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```
Comment (rohanmathur91) [inline src/setupTests.js]: this also you can delete it

**Review by rohanmathur91 — APPROVED** [review]
- The code is nicely structured 
- Added a few cleanups, have a look
- Also one suggestion, try to have a preview link

File: src/App.css (hunk 1 above)
Comment (prakashsakari) [reply src/App.css]: thanks, updated

File: src/index.js (hunk 3 above)
Comment (prakashsakari) [reply src/index.js]: thanks, updated

File: src/reportWebVitals.js (hunk 4 above)
Comment (prakashsakari) [reply src/reportWebVitals.js]: thanks, updated

File: src/setupTests.js (hunk 5 above)
Comment (prakashsakari) [reply src/setupTests.js]: thanks, updated

File: src/components/QuizCard/QuizCard.jsx (hunk 2 above)
Comment (prakashsakari) [reply src/components/QuizCard/QuizCard.jsx]: updated, thanks

**Review by anupkgurung — APPROVED** [review]
nice structure,
try to keep the merge clean (excluding un required files like package.json, .png) then can be pushed once the pull is approved

**Pull request merged**